wcfsetup/install/files/lib/page/AccountSecurityPage.class.php

```diff
@@ -0,0 +1,56 @@
+<?php
+namespace wcf\page;
+use wcf\system\menu\user\UserMenu;
+use wcf\system\session\Session;
+use wcf\system\session\SessionHandler;
+use wcf\system\WCF;
+
+/**
+ * Shows the account security page.
+ *
+ * @author	Joshua Ruesweg
+ * @copyright	2001-2020 WoltLab GmbH
+ * @license	GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @package	WoltLabSuite\Core\Page
+ */
+class AccountSecurityPage extends AbstractPage {
+	/**
+	 * @inheritDoc
+	 */
+	public $loginRequired = true;
+	
+	/**
+	 * @var Session[]
+	 */
+	public $activeSessions;
```

Comment:
```suggestion
	private $activeSessions;
```
I don't want plugins touching this value.